modules/recommended-etcd-practices.adoc

```diff
@@ -6,59 +6,44 @@
-run etcd on machines backed by SSD/NVMe disks with low latency and high throughput.
+.Monitor these key metrics:
 
-Some of the key metrics to monitor on a deployed {product-title} cluster
-are p99 of etcd disk write ahead log duration and the number of etcd leader changes.
-Use Prometheus to track these metrics. `etcd_disk_wal_fsync_duration_seconds_bucket`
-reports the etcd disk fsync duration, `etcd_server_leader_changes_seen_total` reports
-the leader changes. To rule out a slow disk and confirm that the disk is reasonably fast,
-99th percentile of the `etcd_disk_wal_fsync_duration_seconds_bucket` should be less than 10ms.
+* `etcd_server_quota_backend_bytes`, which is the current quota limit
+* `etcd_mvcc_db_total_size_in_use_in_bytes`, which indicates the actual database usage after a history compaction
+* `etcd_debugging_mvcc_db_total_size_in_bytes`, which shows the database size, including free space waiting for defragmentation
 
-Fio, a I/O benchmarking tool can be used to validate the hardware for etcd before or after
-creating the {product-title} cluster. Run fio and analyze the results:
+For more information about defragmenting etcd, see the "Defragmenting etcd data" section.
 
-Assuming container runtimes like podman or docker are installed on the machine under test and
-the path etcd writes the data exists - /var/lib/etcd, run:
+Because etcd writes data to disk and persists proposals on disk, its performance depends on disk performance. Slow disks and disk activity from other processes can cause long fsync latencies. Those latencies can cause etcd to miss heartbeats, not commit new proposals to the disk on time, and ultimately experience request timeouts and temporary leader loss. Run etcd on machines that are backed by SSD or NVMe disks with low latency and high throughput. Consider single-level cell (SLC) solid-state drives (SSDs), which provide 1 bit per memory cell, are durable and reliable, and are ideal for write-intensive workloads.
+
+Some key metrics to monitor on a deployed {product-title} cluster are p99 of etcd disk write ahead log duration and the number of etcd leader changes. Use Prometheus to track these metrics.
+
+* The `etcd_disk_wal_fsync_duration_seconds_bucket` metric reports the etcd disk fsync duration.
+* The `etcd_server_leader_changes_seen_total` metric reports the leader changes.
+* To rule out a slow disk and confirm that the disk is reasonably fast, verify that the 99th percentile of the `etcd_disk_wal_fsync_duration_seconds_bucket` is less than 10 ms.
+
+To validate the hardware for etcd before or after you create the {product-title} cluster, you can use an I/O benchmarking tool called fio. Run fio and analyze the results:
+
+.Prerequisites
+
+* Container runtimes such as Podman or Docker are installed on the machine that you're testing.
+* Data is written to the `/var/lib/etcd` path.
 
 .Procedure
-Run the following if using podman:
+If you use Podman, run this command:
```

Comment:
Single steps need to start with `*`, and I think it would be more effective to make these step options under a  common step.

```
* Run fio and analyze the results: (I am assuming that this is the step. If so, remove this from line 27).
** If you use Podman, run this command:
+
----
command
----

** If you use Docker, run this command:
+
----
command
----
```

Reply:
(You can use the [grandparent list syntax](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#enclose-in-open-block) to attach the note to the first level of the step.)